pinax/stripe/tests/test_webhooks.py

```diff
@@ -344,23 +345,91 @@ def test_plan_created(self, EventMock):
 
 
 class CustomerSubscriptionCreatedWebhookTest(TestCase):
-
     @patch("stripe.Customer.retrieve")
     @patch("pinax.stripe.actions.customers.sync_customer")
     def test_process_webhook(self, SyncMock, RetrieveMock):
-        event = Event.objects.create(kind=CustomerSubscriptionCreatedWebhook.name, customer=Customer.objects.create(), webhook_message={}, valid=True, processed=False)
+        event = Event.objects.create(kind=CustomerSubscriptionCreatedWebhook.name, customer=Customer.objects.create(),
+                                     webhook_message={}, valid=True, processed=False)
         CustomerSubscriptionCreatedWebhook(event).process_webhook()
         self.assertTrue(SyncMock.called)
 
     @patch("pinax.stripe.actions.customers.sync_customer")
     def test_process_webhook_no_customer(self, SyncMock):
-        event = Event.objects.create(kind=CustomerSubscriptionCreatedWebhook.name, webhook_message={}, valid=True, processed=False)
+        event = Event.objects.create(kind=CustomerSubscriptionCreatedWebhook.name, webhook_message={}, valid=True,
+                                     processed=False)
         CustomerSubscriptionCreatedWebhook(event).process_webhook()
         self.assertFalse(SyncMock.called)
 
 
-class InvoiceCreatedWebhookTest(TestCase):
+class CustomerSubscriptionUpdatedWebhookTest(TestCase):
+    WMD = {"object": {"livemode": False, "id": "my_sub", "billing": "charge_automatically",
```

Comment:
Please use a more descriptive name, also for `VMD` below.

Reply:
It's not that i don't agree with you, but jeezzz.... i'm just trying to help here...?

What do you suggest?

Paul